Sanitizer error in ydb/tests/stress/scheme_board/pile_promotion/tests/test_scheme_board_workload.py.TestSchemeBoard.test_scheme_board

ydb/tests/stress/scheme_board/pile_promotion/tests/test_scheme_board_workload.py.TestSchemeBoard.test_scheme_board

**Build Information:**
- Version: 16ccbe8aee73119b6b3bf5af118feba55ba69799
- Build Preset: release-msan
- Branch: prestable-25-3

**Run Command:**
```
./ya make -ttt --build "release" --sanitize="memory" -DDEBUGINFO_LINES_ONLY -F 'test_scheme_board_workload.py::TestSchemeBoard::test_scheme_board' ydb/tests/stress/scheme_board/pile_promotion/tests
```

**Owner:** [TEAM:@ydb-platform/engineering](https://github.com/orgs/ydb-platform/teams/engineering)

**Test run history:** [link](https://datalens.yandex/34xnbsom67hcq?branch=prestable-25-3&full_name=ydb/tests/stress/scheme_board/pile_promotion/tests/test_scheme_board_workload.py.TestSchemeBoard.test_scheme_board&build_type=release-msan)

More info in [dashboard](https://datalens.yandex/4un3zdm0zcnyr)

**Logs:**
- [log](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_Small_and_Medium/17708525640/ya-prestable-25-3-x86-64-msan/try_1/artifacts/logs/ydb/tests/stress/scheme_board/pile_promotion/tests/test-results/py3test/testing_out_stuff/test_scheme_board_workload.py.TestSchemeBoard.test_scheme_board.log)
- [logsdir](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_Small_and_Medium/17708525640/ya-prestable-25-3-x86-64-msan/try_1/test_artifacts/ydb-tests-stress-scheme_board-pile_promotion-tests.zip)

**Error Summary:**
```
E   GRpc memory quota was set but disabled due to issues with grpc quoter, to enable it use EnableGRpcMemoryQuota option
E   ==1275425==WARNING: MemorySanitizer: use-of-uninitialized-value
E   warning: address range table at offset 0x820 has a premature terminator entry at offset 0x830
E   #0 0x44879766 in MakeHolder<NKikimr::NTracing::TOnHandleStateStorageInfoResolve, unsigned int &, unsigned int &, unsigned int> /-S/util/generic/ptr.h:386:27
E   #1 0x44879766 in NKikimr::TTablet::HandleStateStorageInfoResolve(TAutoPtr<NActors::TEventHandle<NKikimr::TEvStateStorage::TEvInfo>, TDelete>&) /-S/ydb/core/tablet/tablet_sys.cpp:847:13
E   #2 0x1be4aa88 in NActors::IActor::Receive(TAutoPtr<NActors::IEventHandle, TDelete>&) /-S/ydb/library/actors/core/actor.cpp:406:17
E   #3 0x1beec10b in NActors::TExecutorThread::Execute(NActors::TMailbox*, bool) /-S/ydb/library/actors/core/executor_thread.cpp:268:28
E   #4 0x1bef5eb7 in NActors::TExecutorThread::ProcessExecutorPool()::$_0::operator()(NActors::TMailbox*, bool) const /-S/ydb/library/actors/core/executor_thread.cpp:458:39
E   #5 0x1bef556c in NActors::TExecutorThread::ProcessExecutorPool() /-S/ydb/library/actors/core/executor_thread.cpp:510:13
E   #6 0x1bef72f2 in NActors::TExecutorThread::ThreadProc() /-S/ydb/library/actors/core/executor_thread.cpp:536:9
E   #7 0x19ac4633 in (anonymous namespace)::TPosixThread::ThreadProxy(void*) /-S/util/system/thread.cpp:245:20
E   #8 0x7fb1a56bcac2  (/lib/x86_64-linux-gnu/libc.so.6+0x94ac2) (BuildId: d5197096f709801829b118af1b7cf6631efa2dcd)
E   #9 0x7fb1a574e84f  (/lib/x86_64-linux-gnu/libc.so.6+0x12684f) (BuildId: d5197096f709801829b118af1b7cf6631efa2dcd)
E   
E   SUMMARY: MemorySanitizer: use-of-uninitialized-value /-S/util/generic/ptr.h:386:27 in MakeHolder<NKikimr::NTracing::TOnHandleStateStorageInfoResolve, unsigned int &, unsigned int &, unsigned int>
E   Exiting

```

--- a/ydb/core/base/statestorage_proxy.cpp
+++ b/ydb/core/base/statestorage_proxy.cpp
@@ -794,6 +794,12 @@ public:
     TStateStorageRingGroupProxyRequest(TIntrusivePtr<TStateStorageInfo> info)
         : Info(info)
         , RingGroupPassAwayCounter(0)
+        , TabletID(0)
+        , Cookie(0)
+        , CurrentGeneration(0)
+        , CurrentStep(0)
+        , Locked(false)
+        , LockedFor(0)
     {
     }
 

